NEWS

```diff
@@ -6,6 +6,21 @@
 ##
 #
 
+## 1.2.0 (June 13, 2017)
+
+Supported platforms
+  - Shared memory: KNEM, CMA, XPMEM, SYSV, Posix
+  - RDMA interconnects with VERBs interface (InfiniBand, RoCE, iWarp, Omni-Path),
+    Cray Gemini, Cray Aries, Mellanox ConnectX4/5
```

Comment:
No need to mention explicitly which Mellanox devices are supported, it falls under "InfiniBand" category